code/__DEFINES/dcs/signals.dm

```diff
@@ -525,3 +525,9 @@
 
 // Used for smothering fires upon weather event start/stop
 #define COMSIG_GLOB_WEATHER_CHANGE "!weather_event_changed"
+
+// sentry laptop
```

Comment:
for the sake of sanity, please add a comment per signal define with any args sent (excluding `datum/source`)